Dockerfile

```diff
@@ -1 +1,16 @@
-��
\ No newline at end of file
+FROM python:3.6-slim-stretch
+
+RUN apt-get update && apt-get install -y git python3-dev gcc \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY requirements.txt .
+
+RUN pip install --upgrade -r requirements.txt
+
+COPY src src/
+
+RUN python -m src.backend_api
+
+EXPOSE 5000
+
+CMD ["python", "-m", "src.backend_api"]
```

Comment:
I'd put this dockerfile and any other files you don't need for heroku in your gitignore until you have *working* support for a local install.